packages/agent-kit/src/adapters/anthropic.ts

```diff
@@ -120,6 +120,11 @@ export const requestParser: AgenticModel.RequestParser<Anthropic.AiModel> = (
 export const responseParser: AgenticModel.ResponseParser<Anthropic.AiModel> = (
   input
 ) => {
+  
+  if (input.type === "error") {
```

Comment:
@jpwilliams throwing an error here works for "Inngest-less" setup. It should not break durability as `step.infer()` should catch this errors at the gateway level and retry them, right?

Reply:
Another design could be to have the `responseParser()` implementation return a `Symbol` that is only thrown in the context of an "Inngest-less" environment, wdyt?

Reply:
This would only ever be catching Inngestless, I believe. As you say, `step.ai.infer()` would retry errors, but then currently when retries are exhausted it will turn that into an [`AIGatewayError`](https://github.com/inngest/inngest/blob/aeb19f60840d1b2ee341066c03b25d1ed754d731/pkg/execution/executor/executor.go#L2232-L2237) and the `step.ai.infer()` call throws.

Am I correct in thinking you're just trying to replicate that behaviour for Inngestless setups? If so, I believe this should do the job correctly, as we'd never expect the result of a `step.ai.infer()` to contain this. 🤔